### PR TITLE
Removing mds-daily

### PR DESCRIPTION
**Pull request by janedotx**
## 📚 Purpose
I've confirmed with @brianngca that mds-daily won't be required in 1.0, as the important functions of mds-daily should be covered by the improved version of the compliance API.

## 📦 Impacts:
* mds-db
* mds-daily

## ✅ PR Checklist
- [ ] Add or remove checklist items to suit your needs

### CODE REVIEW

File: packages/mds-daily/db-helpers.ts [hunk 1]
```diff
@@ -1,159 +0,0 @@
-import logger from '@mds-core/mds-logger'
-import db from '@mds-core/mds-db'
```
Comment (drtyh2o) [inline packages/mds-daily/db-helpers.ts:2]: Can any/all of the db functions be removed?

**Review by drtyh2o — COMMENTED** [review]
Can the container image also be removed?

**Comment by janedotx** [comment]
> Can the container image also be removed?

Yeah, just did it. I thought I was going to have it removed before anyone noticed.

File: packages/mds-daily/db-helpers.ts (hunk 1 above)
Comment (janedotx) [reply packages/mds-daily/db-helpers.ts:2]: I don't understand what you mean by "removed" here. Do you mean, can some of these helpers be taken out and used elsewhere?

Comment (drtyh2o) [reply packages/mds-daily/db-helpers.ts:2]: Just wondering if all the mds-db functions that are _only_ used by mds-daily have been deleted.

Comment (janedotx) [reply packages/mds-daily/db-helpers.ts:2]: Good call. I just checked and there were a few left that I've now removed.

**Review by drtyh2o — APPROVED** [review]
LGTM except for the Jenkins failure.

**Review by avatarneil — APPROVED** [review]

**Comment by janedotx** [comment]
> LGTM except for the Jenkins failure.

All tests pass locally, except for `mds-metrics-sheets`, which is broken in 1.0. The Jenkins failure is due to `yarn audit` failing, but those vulnerabilities exist in the parent branch, and I think fixing them is outside of the scope of this PR.

**Comment by avatarneil** [comment]
@janedotx should wack `mds-metrics-sheet` as part of this too, come to think of it. It depends on `mds-daily` (albeit indirectly via API calls).

**Comment by drtyh2o** [comment]
Should `.iyarc` the vulnerabilities if we’re not fixing them for now. Also, I suspect these branches’ dependencies are very stale.

**Comment by janedotx** [comment]
> Should `.iyarc` the vulnerabilities if we’re not fixing them for now. Also, I suspect these branches’ dependencies are very stale.

Yeah, I want to give `dev/upgrade-to-1.0` some care and feeding.

**Review by avatarneil — APPROVED** [review]

**Pull request merged**